dmv/dmv.js

```diff
@@ -19,13 +19,18 @@ DMV.prototype.currentCustomerFor = function(agent) {
 }
 
 DMV.prototype.nextCustomer = function() {
+  if (this.customers.length === 0) {
```

Comment:
this code is dank.  great work.

Reply:
Im also a big fan of 'if' statements. BIG fan.

Reply:
Please provide another comment to demonstrate "Speak fluently using object-oriented / coding terminology; take code and describe it using words like method, parameter, declare, instantiate...etc"